serenity-model/src/main/java/net/thucydides/core/configuration/SystemPropertiesConfiguration.java

```diff
@@ -61,7 +61,7 @@
 
     protected String defaultBaseUrl;
 
-    private final EnvironmentVariables environmentVariables;
+    protected EnvironmentVariables environmentVariables;
```

Comment:
Changed this to none final so instead, the environment variables can be maintained on the same instance of the configuration